roles/_init/tasks/main.yml

```diff
@@ -38,7 +51,9 @@
     executable: /bin/bash
   delegate_to: "localhost"
   register: _init_vars_dirs_md5
-  with_items: "{{ _init_vars_dirs_list }}"
+  with_items:
+    - "{{ _init_vars_dirs_list }}"
+    - "{{ playbook_dir }}"
```

Comment:
I'm just wondering about moving `playbook_dir` from the `set_fact` to `with_items`, if that will have any undesired side effects? It doesn't get included in the `Include vars directories` task any more either, but that might not matter if Ansible includes the directory by default anyway? Did you actually test the behaviour? If not, might be worth running a proof of concept test playbook to see what the `_init` role does.